globus_sdk/exc.py

```diff
@@ -133,6 +134,30 @@ def _load_from_json(self, data):
         self.request_id = data["request_id"]
 
 
+class AuthAPIError(GlobusAPIError):
+    """
+    Error class for the API components of Globus Auth.
+
+    Customizes JSON parsing.
+    """
+    def _load_from_json(self, data):
+        """
+        Load error data from a JSON document.
+
+        Sets `code` statically because Auth doesn't have API error codes, and
+        looks for a top-level "error" attribute.
+        """
+        self.code = "Error"
```

Comment:
Ah, that explains it! I saw things that looked like the `reason`, because that's how a lot of the codes read.

@aaschaer, please scratch that work. I'll fix my PR to not assume that `code` is absent, as you suggested originally.